kapajira/kapacitor/utils.py

```diff
@@ -0,0 +1,34 @@
+import json
+from collections import abc
+
+
+class AlertDataParser:
+    @classmethod
+    def parse(cls, alert_data):
+        alert_dict = json.loads(alert_data)
+        return AlertData(alert_dict)
+
+
+class AlertData:
+    """ A read-only façade for navigating a AlertNode JSON object
+        using attribute notation
+    """
+
+    def __init__(self, mapping):
+        self._data = dict(mapping)
+
+    def __getattr__(self, name):
+        if hasattr(self._data, name):
```

Comment:
Actually it should not be possible as mapping/list are in fact instances of AlertData, and list of alertdata instances. But I've added one more thing so attributes cannot be changed.